app/src/main/java/io/rapidpro/androidchannel/SyncHelper.java

```diff
@@ -51,23 +35,12 @@
 import io.rapidpro.androidchannel.payload.SyncPayload;
 import io.rapidpro.androidchannel.util.Http;
 
-/**
- * Syncs our messages with the server.
- */
-public class SyncService extends JobIntentService {
-
-    /**
-     * Unique job ID for this service.
-     */
-    public static final int JOB_ID = 1001;
-
-    /**
-     * Convenience method for enqueuing work in to this service.
-     */
-    static void enqueueWork(Context context, Intent work) {
-        enqueueWork(context, SyncService.class, JOB_ID, work);
-    }
+import static android.content.Context.CONNECTIVITY_SERVICE;
 
+public class SyncHelper {
+
+    // how many seconds between polling
+    public static final long POLLING_INTERVAL = 60;
```

Comment:
I think 60 secs is a little aggressive, what about 5 min or 10 min? @ericnewcomer @nicpottier